libshvbroker/src/brokerapp.h

```diff
@@ -77,6 +77,10 @@ class SHVBROKER_DECL_EXPORT BrokerApp : public QCoreApplication
 	rpc::WebSocketServer* webSocketServer();
 #endif
 
+#ifdef WITH_SHV_LDAP
+	void setGroupForLdapUser(const std::string_view& user_name, const std::string_view& group_name);
```

Comment:
What about to save current user LDAP groups in ClientConnection. They can be sent by `Q_SIGNAL AuthThread::resultReady(login_result, user_groups)`

Reply:
I'm saving it in BrokerApp which is the only place where it's used afterwards. What benefits would saving it in ClientConnectionOnBroker give? In `accessGrantForRequest`, the normal shv user groups are also fetched by name and from the AclManager, not from the Connection class, so I thought I'd keep it consistent. Although I'm not against making another patch that saves both things in the Connection.